### PR TITLE
Sets up GA to track recurring donations distinct from one-off gifts

### PR DESCRIPTION
**Pull request by adamlofting**

### CODE REVIEW

File: components/simple-paypal.jsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ var simplePaypal = React.createClass({
           <input type="hidden" name="cmd" value="_xclick-subscriptions"/>
           <input type="hidden" name="business" value="44ZHAVWJHTK2N"/>
           <input type="hidden" name="lc" value={this.props.paypalLocal}/>
-          <input type="hidden" name="item_name" value="Mozilla Foundation Recurring Donation"/>
+          <input type="hidden" name="item_name" value="Mozilla Foundation Monthly Donation"/>
```
Comment (adamlofting) [inline components/simple-paypal.jsx]: I thought we should be specific in the name about how often the donation occurs.

File: public/js/analytics.js [hunk 2]
```diff
@@ -32,15 +43,37 @@
   var str_tx_re = /str_id=(\w+)/;
   var str_amt_re = /str_amount=([\w\.]+)/;
   var str_cc_re = /str_currency=(\w+)/;
+  var str_frequency_re = /str_frequency=(\w+)/;
```
Comment (adamlofting) [inline public/js/analytics.js]: We're not actually passing this param through yet for Stripe recurring, but the GA code is ready to go when Stripe is.

**Comment by adamlofting** [comment]
This finishes last item on #16

File: components/simple-paypal.jsx (hunk 1 above)
Comment (alicoding) [inline components/simple-paypal.jsx]: I like Monthly vs Recurring here. :+1:

**Comment by alicoding** [comment]
r+

**Pull request merged**